data/src/main/java/com/example/data/mappers/WeatherMapper.kt

```diff
@@ -0,0 +1,85 @@
+package com.example.data.mappers
+
+import com.example.data.service.responses.CloudResponse
+import com.example.data.service.responses.WeatherResponse
+import com.example.data.service.responses.MainResponse
+import com.example.data.service.responses.ListResponse
+import com.example.data.service.responses.ResponseApi
+import com.example.data.service.responses.CityResponse
+import com.example.data.service.responses.CoordResponse
+import com.example.data.service.responses.RainResponse
+import com.example.data.service.responses.SysResponse
+import com.example.data.service.responses.WindResponse
+import com.example.domain.entities.ResponseWeatherInfo
+import com.example.domain.entities.MainWeather
+import com.example.domain.entities.City
+import com.example.domain.entities.Cloud
+import com.example.domain.entities.Coordinates
+import com.example.domain.entities.ListWeather
+import com.example.domain.entities.Rain
+import com.example.domain.entities.Sys
+import com.example.domain.entities.Wind
+import com.example.domain.entities.WeatherDescription
+
+fun ResponseApi.transformApi() = ResponseWeatherInfo(
```

Comment:
it is okay that all these mappers be public?

Reply:
Not all, only one is public, done :)